src/gluonnlp/data/batchify/batchify.py

```diff
@@ -340,3 +340,33 @@ def __call__(self, data):
         for i, ele_fn in enumerate(self._fn):
             ret.append(ele_fn([ele[i] for ele in data]))
         return tuple(ret)
+
+class List(object):
+    """Simply forward the list of input data.
+
+    This is particularly useful when the Dataset contains textual data
+    and in conjonction with the `Tuple` batchify function.
+
+    Examples
+    --------
+    >>> a = ([1, 2, 3, 4], "I am using MXNet")
+    >>> b = ([5, 7, 2, 5], "Gluon rocks!")
+    >>> c = ([1, 2, 3, 4], "Batchification!")
+    >>> _, l = gluonnlp.data.batchify.Tuple(gluonnlp.data.batchify.Stack(),
+    ...                                     gluonnlp.data.batchify.List())([a, b, c])
+    >>> print(l)
+    ['I am using MXNet', 'Gluon rocks!', 'Batchification!']
+    """
+    def __call__(self, x):
```

Comment:
I think we assumed that input of a batchify function needs to be a tuple.

We should use (self, *args) instead of (self, x)

Reply:
The other batchify functions use (self, x) as the signature so we should stick to it. Ignore this comment.